packages/yourloops/test/integration/assert/footer.ts

```diff
@@ -67,3 +57,35 @@ export const checkFooter = ({ role, needFooterLanguageSelector }: CheckFooterPro
   expect(releaseNotesLink).toHaveAttribute('href', 'fake-urlyourloops-release-notes.pdf')
   expect(contactLink).toBeVisible()
 }
+
+export const checkFooterForUserNotLoggedIn = (hasLanguageSelector: boolean = false) => {
+  const footer = within(screen.getByTestId('footer'))
+  const trainingLink = footer.getByText('Training')
+  expect(trainingLink).toBeVisible()
+  expect(trainingLink).toHaveAttribute('href', diabeloopExternalUrls.training())
+  checkFooter(hasLanguageSelector)
+}
+
+export const checkFooterForHcp = (hasLanguageSelector: boolean = false) => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Given how the implementation of `checkFooterForHcp()`, `checkFooterForPatient()` and `checkFooterForCaregiver()` look alike, it might be interesting to factorize the code in only one method?

(it may even be possible to factorize with `checkFooterForUserNotLoggedIn()` but I let you decide whether it's relevant or not)

```ts
export const checkFooter = (role: UserRole, hasLanguageSelector: boolean = false) => {
  const footer = within(screen.getByTestId('footer'))
  const trainingLink = footer.getByText('Training')
  expect(trainingLink).toBeVisible()
  expect(trainingLink).toHaveAttribute('href', diabeloopExternalUrls.training(role))
  checkFooter(hasLanguageSelector)
}
```